File: dataset/java2python/java/FRIENDS_PAIRING_PROBLEM_2.java
int f_gold ( int n ) {
  int a = 1 , b = 2 , c = 0 ;
  if ( n <= 2 ) {
    return n ;
  }
  for ( int i = 3 ;
  i <= n ;
  i ++ ) {
    c = b + ( i - 1 ) * a ;
    a = b ;
    b = c ;
  }
  return c ;
}

File: result/repair/transcoder/java2python/python_repair/FRIENDS_PAIRING_PROBLEM_2.py
def f_gold ( n ) :
    a , b , c = 1 , 2 , 0
    if n <= 2 :
        return n
    i=3 
    while i<n+1 :
        c = b + ( i - 1 ) * a
        a , b , c = b , c , c
        i += 1
    return c

